test/integration/features/basic.feature

```diff
@@ -35,34 +35,25 @@ Feature: Basic test
         And stdout should contain "Setting up KVM"
         And stdout should contain "Installing libvirt service and dependencies" 
         And stdout should contain "Adding user to libvirt group"
-        And stdout should contain "Will use root access: add user to libvirt group"
         And stdout should contain "Enabling libvirt"
         And stdout should contain "Starting libvirt service"
-        And stdout should contain "Will use root access: start libvirtd service"
```

Comment:
I don't understand why these would not be part of stdout?
```
INFO Starting libvirt 'crc' network               
INFO Checking if NetworkManager is installed      
INFO Checking if NetworkManager service is running 
INFO Writing Network Manager config for crc       
INFO Writing dnsmasq config for crc               
INFO Will use root access: write dnsmasq configuration in /etc/NetworkManager/dnsmasq.d/crc.conf 
INFO Will use root access: execute systemctl daemon-reload command 
INFO Will use root access: execute systemctl stop/start command 
```
This is what I get with a `crc setup` on my test machine.

Reply:
@cfergeau it looks like the output does not contain the `Will use root access` even on CI (see CentosCI console output). So, I guess, it'll be necessary to remove the `/etc/NetworkManager/dnsmasq.d/crc.conf` file before the feature if we want to test it from fresh.

Reply:
if you look at https://github.com/code-ready/crc/pull/587 , CI was successful and the integration tests have "Will use root access" https://github.com/cfergeau/crc/blob/win-lint/test/integration/features/basic.feature so I don't understand what's going on :-/
Removing `/etc/NetworkManager/dnsmasq.d/crc.conf` just causes one of the tests to show the "Will use root access" message, not all of them :(

Reply:
> Removing /etc/NetworkManager/dnsmasq.d/crc.conf just causes one of the tests to show the "Will use root access" message, not all of them :(
This would be enough, right? We want to test that line of execution at least once, so exactly once would be good I guess.

I'm a fair bit confused too! Will look around a bit more for some nicer solution.

Reply:
> This would be enough, right? We want to test that line of execution at least once, so exactly once would be good I guess.

We also want to test `crc setup` on a blank machine, I was thinking centos CI would do that. If you tell me the "Will use root access:" lines need to be removed, that means we are not testing at all the host machine configuration `crc` is supposed to be doing.
If there is one `crc setup` test checking this, and this is about running `crc setup` a second time on the same host, then it's fine.

Reply:
@cfergeau sorry, I'm confused. I want to remove the file `/etc/NetworkManager/dnsmasq.d/crc.conf` before the testsuite, not the lines containing `Will use root access:` from the test itself. Then inside the tests, the first time `crc setup` is executed, I'll expect `Will use root access:` lines. The second (and later) time `crc setup` is executed in the tests, I will not expect the `Will use root access:` lines. Which part of configuration that `crc` is supposed to be doing wouldn't be tested this way? Cheers!

Reply:
The very first time `crc setup` is run, you should get `Will use root access:` lines even without removing `/etc/NetworkManager/dnsmasq.d/crc.conf` before. If you need to remove it, you'll likely only get one such line, rather than multiple ones.
As I was saying in https://github.com/code-ready/crc/pull/602#discussion_r326082144 , I don't think this is needed as we have CI currently passing with the integration test checking for "Will use root access:" and not complaining about it.
So I don't understand what changed and what we are discussing exactly :-/

Reply:
Cool, I just assumed that "The very first time" means literally only once on each machine. Then simulating "the very first time" by removing the file before the test would make sense.

> If you need to remove it, you'll likely only get one such line, rather than multiple ones.

I got all three when I checked this. Hmmm...

> we have CI currently passing with the integration test checking for "Will use root access:" and not complaining about it.

Agreed, that's the puzzle now.

Reply:
> Cool, I just assumed that "The very first time" means literally only once on each machine. Then simulating "the very first time" by removing the file before the test would make sense.

Just one more clarification on this. Each preflight check/fix is independent, if you remove `/etc/NetworkManager/dnsmasq.d/crc.conf` then the preflight check for this file existence will fail, and the fix will run. Other unrelated checks/fixes won't be run (for example, the codepath doing "sudo systemctl enable libvirtd" won't be exercised).

> 
> > If you need to remove it, you'll likely only get one such line, rather than multiple ones.
> 
> I got all three when I checked this. Hmmm...
> 
> > we have CI currently passing with the integration test checking for "Will use root access:" and not complaining about it.
> 
> Agreed, that's the puzzle now.

Reply:
so machines aren't really clean?

Reply:
@cfergeau @gbraad I haven't encountered a case where `Will use root access:` wasn't part of the output. So the issue seems to have gone away for the time being. This also makes the changes that this PR introduces much smaller - but there are some.